pkg/controller/build/build_controller.go

```diff
@@ -148,20 +151,22 @@ func (r *ReconcileBuild) Reconcile(request reconcile.Request) (reconcile.Result,
 
 	// Validate if the build strategy is defined
 	if b.Spec.StrategyRef != nil {
-		if err := r.validateStrategyRef(ctx, b.Spec.StrategyRef, b.Namespace); err != nil {
-			ctxlog.Error(ctx, err, "failed validating the strategy reference", "Build", b.Name)
+		err := r.validateStrategyRef(ctx, b.Spec.StrategyRef, b.Namespace)
```

Comment:
Why we change this to two lines?

Reply:
no reason, I rollback to the previous state, good catch